RobotsAndPencils/react-gantry
# SVG Webpack Setup

This builds off of PR #2 and provides an answer to Issue #3.

Using a combination of [Rule.oneOf](https://webpack.js.org/configuration/module/#rule-oneof) and [Rule.resourceQuery](https://webpack.js.org/configuration/module/#rule-resourcequery) from webpack we can make importing SVGs default to the file loader, but with a `?inline` flag on the `import` path use the react-svg-loader instead.

No unnecessary splitting of directories, totally reusable files, just a different way to import as demonstrated in the `/src/components/profile/profile.js` file.

## To Test
Pull, `npm run start` and observe that the SVGs on the Profile component both work.
`npm run build` should pull the svg file into the bundle because it's needed for the `<img>` usage.
